images/tier1/down.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="18" width="18" viewBox="0 0 18 18">
```

Comment:
It'd be good to have these arrows grouped like the existing arrows and the chevrons. Might I suggest `arrow-thin-down`, `arrow-thin-up`, `arrow-thin-left` and `arrow-thin-right`?